modules/nw-dns-forward.adoc

```diff
@@ -41,34 +41,37 @@ spec:
     zones: <2>
       - foo.com
     forwardPlugin:
-      upstreams: <3>
+      policy: Random <3>
+      upstreams: <4>
         - 1.1.1.1
         - 2.2.2.2:5353
   - name: bar-server
     zones:
       - bar.com
       - example.com
     forwardPlugin:
+      policy: Random
```

Comment:
The policy looks good, but could you help update the indention for `zones` and `upstreams` here as well? They can work indeed but don't follow the yaml format. A good example looks like
```
  servers:
  - forwardPlugin:
      policy: Random
      upstreams:
      - 1.1.1.1                               <no indention>
    name: foo-server
    zones:
    - foo.com                                 <no indention>      
```